include/control_table.h

```diff
@@ -0,0 +1,194 @@
+#pragma once
+
+#include <cstdint>
+
+// Defined in linker file
+extern uint8_t __CONTROL_TABLE_START;
+
+enum class CT {
```

Comment:
ahhh, enum class. Forgot about those. Python doesn't have enums.

Reply:
Looks good, where/when does the flash get read into memory? Also, we are not writing to flash for frequent writes, like goal position, are we?